eitprocessing/binreader/sequence.py

```diff
@@ -70,6 +72,7 @@ class Sequence:
 
     path: Path | str | List[Path | str] = None
     vendor: Vendor = None
+    label: str = None
```

Comment:
```suggestion
    label: str | None = None
```

or

```suggestion
    label: Optional[str] = None 
```

`from typing import Optional`